inference-engine/src/gna_plugin/CMakeLists.txt

```diff
@@ -29,17 +29,20 @@ endif()
 
 #
 # Shared plugin library
-# 
+#
 
 ie_add_plugin(NAME ${TARGET_NAME}
               DEVICE_NAME "GNA"
               SOURCES ${SOURCES} ${HEADERS})
 
+# Enable support of CC for the plugin
+ie_mark_target_as_cc(${TARGET_NAME})
+
 # saving rpath to GNA shared library be used by CI
 log_rpath_from_dir(GNA ${libGNA_LIBRARIES_BASE_PATH})
 
 target_link_libraries(${TARGET_NAME} PRIVATE inference_engine inference_engine_legacy inference_engine_transformations
-        Threads::Threads libGNA)
+        Threads::Threads libGNA openvino::itt)
```

Comment:
If I remember right, if you mark target as `cc` you don't need to add `openvino::itt` library.